AEPIdentity/Tests/IdentityFunctionalTests.swift

```diff
@@ -51,6 +73,61 @@ class IdentityFunctionalTests: XCTestCase {
         XCTAssertNotNil(sharedState?[IdentityConstants.EventDataKeys.VISITOR_ID_ECID])
     }
 
+    /// Tests that a sync event dispatches a shared state update
+    func testSyncIdentifiersForceSyncEventInstallScenario() {
+        // setup
+        setDefaultSuccessNetworkResponse()
+        let configSharedState = [IdentityConstants.Configuration.GLOBAL_CONFIG_PRIVACY: PrivacyStatus.optedIn.rawValue, IdentityConstants.Configuration.EXPERIENCE_CLOUD_ORGID: "test-org-id"]
+        let data = [IdentityConstants.EventDataKeys.IS_SYNC_EVENT: true]
+        let event = Event(name: "Sync Event", type: EventType.identity, source: EventSource.requestIdentity, data: data)
```

Comment:
This input event seems invalid, can we have two tests instead:
1. input is getExperienceCloudID event https://github.com/adobe/aepsdk-core-ios/blob/main/AEPIdentity/Sources/Identity%2BPublicAPI.swift#L85-L88, output ECID returned and force sync network request sent
2. input is a valid syncIdentifier event - https://github.com/adobe/aepsdk-core-ios/blob/main/AEPIdentity/Sources/Identity%2BPublicAPI.swift#L128-L137, output first network request was sent for force sync with ecid, second request was sent with the new identity passed with the syncIdentifier event